<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>905</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="log_in_frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>341</width>
      <height>161</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="usr_login_pushButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>115</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>登录</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="pwd_textEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>密码：</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="usn_textEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>用户名：</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="query_frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>341</width>
      <height>121</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>商铺链接：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="query_fans_pushButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>115</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>查看粉丝</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="store_link_textEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="flw_fans_pushButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>70</y>
       <width>115</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>关注粉丝</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="fans_frame">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>481</width>
      <height>531</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTableWidget" name="fans_tableWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>41</y>
       <width>441</width>
       <height>421</height>
      </rect>
     </property>
     <!-- <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property> -->
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>271</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>粉丝列表</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="login_status_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>你好，访客！</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>905</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHello">
    <property name="title">
     <string>main</string>
    </property>
   </widget>
   <addaction name="menuHello"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
